roles/zabbix_javagateway/tasks/Debian.yml

```diff
@@ -29,17 +29,41 @@
   become: yes
   when:
     - ansible_distribution == "Debian"
+    - ansible_machine == "aarch64"
```

Comment:
@dj-wasabi is this typo? Locking both this and the following tasks to `ansible_machine == "aarch64"`